src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,7 +95,13 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        if (someDate.isBefore(LocalDate.now())) {
```

Comment:
let's create separate variable for LocalDate.now(), because now we are calling it multiple times

Reply:
Ok)